packages/ui/src/components/Command/Command.utils.tsx

```diff
@@ -358,14 +358,14 @@ export function useHistoryKeys({ enable, messages, setPrompt }: UseHistoryKeysOp
         case 'ArrowUp':
           setMessageSelectionIndex((index) => {
             const newIndex = Math.max(index - 1, 0)
-            setPrompt(messages[newIndex] ?? '')
+            messages[newIndex] ? setPrompt(messages[newIndex]) : null
```

Comment:
Since there's no assignment, lets use an `if` statement for better clarity.

Reply:
Hi @gregnr I've pushed the changes. Thanks for the feedback!